dev/dune-dbg.in

```diff
@@ -16,4 +16,11 @@ esac
 
 emacs="${INSIDE_EMACS:+-emacs}"
 
-ocamldebug $emacs $(ocamlfind query -recursive -i-format coq.top_printers) -I +threads -I dev $exe "$@"
+# Note: when doing a local dune build, ocamlfind won't work unless you do
+# export OCAMLPATH=<coq_root>/_build/install/default/lib
+# dune exports this automatically to child processes (e.g. this script).
```

Comment:
The right way to call this script is using `dune exec -- dev/dune-dbg` , IMO I would remove that advice as it is not complete. Also, using the above form will take care of the dependencies.

Reply:
I put in the comment because it confused my that I can't play with ocamlfind on the command line alone - in order to find out what to put in there. Since the ourcome of this is not trivial, I thought others might want to try ocamlfind. This just explains how to do it. Btw.: dune exec doesn't seem to be documented.

Reply:
> Btw.: dune exec doesn't seem to be documented.

`man dune` correctly documents it.

> I put in the comment because it confused my that I can't play with ocamlfind on the command line alone - in order to find out what to put in there. Since the ourcome of this is not trivial, I thought others might want to try ocamlfind

IMHO this kind of advice belongs in the general dune readme, not here. One the basics of dune is that commands run in a sandboxed environment so indeed you need to use `dune exec` or other means to properly run them.

The advice you give above is incorrect as for example you are not setting PATH and other variables, so for example the script won't work if dune wants to use a different switch, or if further composition is going on.